### ISSUE TITLE
libffi 3.0.13-0 includes broken libtool ".la" file on linux-64

### ISSUE THREAD

**Issue by pkgw** [ContinuumIO/anaconda-issues#596]
The current 3.0.13-0 version of the linux-64 `libffi` package includes a Libtool library information file at `lib/libffi.la`. Unfortunately the file embeds an inaccurate path (pointing to a `lib64` directory rather than `lib`) that breaks builds for any libtool-using package that wants to link against the (Ana)Conda version of `libffi`.

The easiest solution is probably just to delete the `.la` file from the package since I don't think they do any good on the supported platforms.

This issue is a duplicate of [conda/conda-recipes#272](https://github.com/conda/conda-recipes/issues/272) but no one seems to have paid attention to that one, and I'm told that this is the proper venue for filing issues along these lines.

**Comment by pkgw** [comment]
I'll add another, less important note from my original bug report that I forgot about:

> Second, some packages seem to assume that libffi will provide a pkg-config file, but the conda package doesn't provide it. This can be worked around, but is annoying. This file has been in the source tree for a long time, so I'm not sure why it's not appearing in the Conda build.

**Comment by kalefranz** [comment]
@ilanschnell Can you also comment here?

**Comment by jakirkham** [comment]
Not sure about `defaults`, but the conda-forge package for `libffi` appears to be fixing the paths correctly. I can tell as it has the line ( `libdir='/opt/anaconda1anaconda2anaconda3/lib'` ). Maybe worth taking a look at how we build it or simply using the recipe/package as appropriate. ( https://github.com/conda-forge/libffi-feedstock )

**Comment by nehaljwani** [comment]
This issue has been fixed in `libffi 3.2.1-1`
```bash
(596) [nwani@centos6x64 ~]$ conda list libffi
# packages in environment at /home/nwani/m3/envs/596:
#
libffi                    3.2.1                         1
(596) [nwani@centos6x64 ~]$ grep libdir $CONDA_PREFIX/lib/libffi.la
libdir='/home/nwani/m3/envs/596/lib'
```
